src/sentry/api/endpoints/project_keys.py

```diff
@@ -81,6 +83,8 @@ def post(self, request, project):
             key = ProjectKey.objects.create(
                 project=project,
                 label=result.get('name'),
```

Comment:
When doing this, I also noticed that this accepts `name` for the input to create, but yet on the serializer, we spit back `label`. :( Not sure if we want to change this to accept `label` instead or not.